opencti-platform/opencti-graphql/src/modules/publicDashboard/publicDashboard-utils.ts

```diff
@@ -106,7 +106,8 @@ export const getWidgetArguments = async (
     ...platformUser,
     origin: { user_id: platformUser.id, referer: 'public-dashboard' },
     allowed_marking: allowedMaxMarkings,
-    capabilities: [accessKnowledgeCapability]
+    capabilities: [accessKnowledgeCapability],
+    inside_platform_organization: true,
```

Comment:
Why hard coding the value to `true` ? Can't we use the value of the `platformUser` ?